### DISCUSSION TITLE
Restrict Read from Database depending on stripe subscription

### DISCUSSION THREAD

**Discussion by mwolf1989** [supabase/supabase#22512] [Questions]
Hi I followed the NexJS + Supabase + Stripe tutorial and wonder how i could restrict read access to tables. 

Should i put the subscription type into the jwt token and check on this using auth.jwt() or it there a different way to archive this?

Best Regards

Marcel

**Comment by GaryAustin1** [comment]
I don't know that guide.  If there is a user to subscription type table then just do a join select in RLS on that table with the user id.

https://supabase.com/docs/guides/auth/row-level-security#using-joins
